controllers/etcdpeer_controller.go

```diff
@@ -331,6 +338,57 @@ func (r *EtcdPeerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, nil
 	}
 
+	if !peer.ObjectMeta.DeletionTimestamp.IsZero() {
```

Comment:
Add a quick comment here "Check if the peer has been marked for deletion"? Might be trivial but it'll help scanning this area of code in future 😄

Reply:
Done.